riocli/configtree/import_keys.py

```diff
@@ -59,43 +65,59 @@ def import_keys(
         update_head: bool,
         milestone: Optional[str],
         export_directory: Optional[str],
+        export_format: Optional[str],
         etcd_endpoint: Optional[str],
         etcd_port: Optional[int],
         etcd_prefix: Optional[str],
+        overrides: Optional[Iterable[str]],
         with_org: bool,
         spinner: Yaspin,
 ) -> None:
-    """
-    Imports keys in a Config tree from YAML files.
-    """
+    """Imports keys from JSON or YAML files.
 
-    data = {}
+    The import command can import keys into an existing Config Tree or
+    to an ETCD cluster. It can also output the keys to files in JSON or
+    YAML format. The command can also apply overrides to the keys before
+    importing them.
 
-    for f in files:
-        file_prefix = Path(f).stem
-        file_format = 'yaml'
-        if f.endswith('json'):
-            file_format = 'json'
+    The keys are imported from the one or more files provided as arguments.
+    The supported file formats are JSON and YAML.
 
-        data[file_prefix] = benedict(f, format=file_format)
-        spinner.write(
-            click.style(
-                '{} File {} processed.'.format(Symbols.SUCCESS, f),
-                fg=Colors.CYAN,
-            )
-        )
+    Usage Examples:
 
-    data, metadata = split_metadata(data)
+      Import keys from master.json to the sootballs Config Tree along with overrides. Also, commit and update the head of the Config Tree.
+
+      $ rio configtree import sootballs master.json --override overrides.json --commit --update-head
+
+      Import keys from master.json to etcd along with overrides.
+
+      $ rio configtree import sootballs master.json --override overrides.json --etcd-endpoint localhost
+
+
+    You can specify more than one override file by providing the files with --override flag multiple times.
+
+    When importing to ETCD, the name of the base JSON or YAML file will be prefixed to the keys.
+
+    Note: If --etcd-endpoint is provided, the keys are imported to the local etcd cluster instead of the rapyuta.io cloud.
+    """
+    data, metadata = _process_files_with_overrides(files, overrides, spinner)
 
     if export_directory is not None:
-        export_to_files(base_dir=export_directory, data=data)
+        export_to_files(base_dir=export_directory, data=data, file_format=export_format)
```

Comment:
suggestion: Since we added the log line for Etcd, can we do the same for export_to_files as well?

Reply:
Done.